generators/upgrade/index.js

```diff
@@ -224,7 +224,7 @@ module.exports = UpgradeGenerator.extend({
         updateJhipster: function() {
             this.log(chalk.yellow('Updating ' + GENERATOR_JHIPSTER + '. This might take some time...'));
             var done = this.async();
-            shelljs.exec('npm install ' + GENERATOR_JHIPSTER, {silent:true}, function (code, msg, err) {
+            shelljs.exec('npm install ' + GENERATOR_JHIPSTER + '@' + this.latestVersion, {silent:true}, function (code, msg, err) {
```

Comment:
@lordlothar99 let's do it in this PR -> you'll see it's quite easy, it has basically the same commands as NPM
And for Webpack, it is run from Yarn or NPM, so that's also very easy to run
(of course, don't have a look at how it works internally, that's VERY complex!!)